internal/config/config.go

```diff
@@ -37,7 +37,7 @@ type MetricsConfig struct {
 }
 
 type NodeConfig struct {
-	Network      string `yaml:"network" envconfig:"NETWORK"`
+	Network      string `yaml:"network" envconfig:"CARDANO_NETWORK"`
```

Comment:
This parameter isn't documented at all in our config example. If you don't do it here, you should create a follow-up issue for it.

Reply:
Filed #28